docs/node/mainnet/eden-upgrade.md

```diff
@@ -209,7 +209,7 @@ Ubuntu versions (or other distributions using older system libraries).
 
 11. Start your node and re-enable auto-starting via your process manager.
 
-[Oasis CLI]: ../../general/manage-tokens/cli/
+[Oasis CLI]: ../../../general/manage-tokens/cli/
```

Comment:
That seems wrong :thinking: 

This should work:

```suggestion
[Oasis CLI]: ../../general/manage-tokens/cli/README.md
```

Reply:
But `../../../general/manage-tokens/cli/` works on the preview: https://deploy-preview-613--trusting-archimedes-14c863.netlify.app/node/mainnet/eden-upgrade

Reply:
Referencing the markdown files instead of URLs is a better idea because it's easier to refactor, links will also work in Github and various IDEs.

Reply:
Also `../../general/manage-tokens/cli/README.md` will go to https://docs.oasis.io/node/general/manage-tokens/cli/README.md that does not exists.